Makefile

```diff
@@ -1,8 +1,9 @@
 CMD_FILES = $(wildcard cmd/wal-g/*.go)
 PKG_FILES = $(wildcard internal/**/*.go internal/**/**/*.go internal/*.go)
 TEST_FILES = $(wildcard test/*.go testtools/*.go)
+PKG := github.com/wal-g/wal-g
```

Comment:
Remove alpine build changes. Because that will be a different PR in wal-g. 

xref: https://github.com/kubedb/wal-g/commit/14ba013e6ce065e649b3818a1dcd6141ae8b3f37